### PR TITLE
Tweaked README for formatting, etc.

### PR DESCRIPTION
**Pull request by GLMeece**
Minor fixes to formatting so underscores don't show when they shouldn't, minor grammatical edits, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,87 +4,74 @@
...
-| Proxy Requests            |                                  |                     |                                |
-|                           | ${proxies}=                      | Create Dictionary   | http=http://acme.com:912       | https=http://acme.com:913 |
-|                           | Create Session                   | github              | http://api.github.com          | proxies=${proxies}    |
-|                           | ${resp}=                         | Get Request         | google                         | /                    |
-|                           | Should Be Equal As Strings       | ${resp.status_code} | 200                            |                      |
-
+|                  |                            |                     |                          |                           |
+| ---------------- | -------------------------- | ------------------- | ------------------------ | ------------------------- |
+| *** Settings *** |                            |                     |                          |                           |
```
Comment (bulkan) [inline README.md]: something is wrong with the formatting of the table

Comment (GLMeece) [inline README.md]: I am trying to see what issue you might be seeing. Here's my final version, which Github seems to render just fine, other than a premature wrap in the first column of the second table.

https://github.com/GLMeece/robotframework-requests/blob/patch-2/README.md

Comment (nizish) [inline README.md]: `Request` line names the session as `google`. Shouldn't that be `github`?

Comment (GLMeece) [inline README.md]: @nizish - my edits didn't change content per se, and this is from the original. However, I think you are correct - it should say `github`.

**Pull request merged**